### PR TITLE
Slug formatting + Urls https validation

### PR DESCRIPTION
**Pull request by Fusionqq**
## replace spaces to dash in slug
screen shot 2018-08-05 at 16 33 52

## validate url for `https`
screen shot 2018-08-05 at 16 33 22

### CODE REVIEW

File: components/articles/EditArticleForm/EditLocaleForm.js [hunk 1]
```diff
@@ -33,19 +34,17 @@ const localeFields = [
...
+  Object.keys(locales).reduce((errors, loc) => {
+    localeFields.forEach(({ field, validator }) => {
+      const path = `${loc}.${field}`;
+      const validationError = validator(get(locales, path));
+
+      set(errors, path, validationError);
+    });
+
+    return errors;
+  }, {});
```
Comment (Drapegnik) [inline components/articles/EditArticleForm/EditLocaleForm.js:53]: in previous implementation `localesValidator` return `null` if there is no errors

now it returns `{}`

is it broke something?

File: components/articles/EditArticleForm/EditLocaleForm.js [hunk 2]
```diff
@@ -92,6 +91,9 @@ const EditLocaleForm = ({ article, prefix, formApi, onRemove, pending, errors })
           formApi={formApi}
           id={`${prefix}.slug`}
           field={`${prefix}.slug`}
+          onChange={slugValue =>
+            formApi.setValue(`${prefix}.slug`, replaceSpaceAndUnderscoreToDash(slugValue))
+          }
```
Comment (Drapegnik) [inline components/articles/EditArticleForm/EditLocaleForm.js]: what about more general `replaceToDash([' _'], slugValue)`?

**Comment by Drapegnik** [comment]
[for locale slug field add space-to-dash replacement on change](https://trello.com/c/UL6RdGHP/260-for-locale-slug-field-add-space-to-dash-replacement-on-change)

File: components/articles/EditArticleForm/EditLocaleForm.js (hunk 2 above)
Comment (Fusionqq) [reply components/articles/EditArticleForm/EditLocaleForm.js]: Nice

File: components/articles/EditArticleForm/EditLocaleForm.js (hunk 1 above)
Comment (Fusionqq) [reply components/articles/EditArticleForm/EditLocaleForm.js:53]: No. Now it returns object like `{
 be: {
  slug: null,
 }
}`

File: components/articles/EditArticleForm/EditLocaleForm.js (hunk 2 above)
Comment (Drapegnik) [reply components/articles/EditArticleForm/EditLocaleForm.js]: or, mb just `formatSlug` and also and slashes replacement

Comment (Fusionqq) [reply components/articles/EditArticleForm/EditLocaleForm.js]: @Drapegnik Are you sure, that we will use this func only for slug in the future?

**Review by Drapegnik — DISMISSED** [review]

**Comment by Drapegnik** [comment]
[to add validation all uploaded images have https:// prefix (with S letter)](https://trello.com/c/rWDBPuax/257-to-add-validation-all-uploaded-images-have-https-prefix-with-s-letter)

**Comment by Drapegnik** [comment]
[add semantic-release & commit-lint](https://trello.com/c/ha9MGzjq/259-add-semantic-release-commit-lint)

**Comment by Drapegnik** [comment]
[investigate slow pages build in dev mode after next update to 6](https://trello.com/c/U7PR3sB5/232-investigate-slow-pages-build-in-dev-mode-after-next-update-to-6)

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**